### ISSUE TITLE
Feedback: 'Transport support'

### ISSUE THREAD

**Issue by selism** [Particular/docs.particular.net#6552]
Feedback for 'Transport support' https://docs.particular.net/servicecontrol/transports
Location in GitHub: https://github.com/Particular/docs.particular.net/blob/master/servicecontrol/transports.md

Please add info on how to specify the schema and catalog for the subscription table when using SQL Transport.

**Comment by helenktsai** [comment]
Hi @selism, we have some [documentation showing schema and catalog](https://docs.particular.net/servicecontrol/transports#transport-specific-features-sql) on that page. Is there additional information you expect to see there?

**Comment by selism** [comment]
For subscription tables that topic only mentions:
`Subscriptions Table=<subscription_table_name> - Specifies SQL subscription table name.`

In my case, the subscription table is in another schema.  I thought that specifying `[schema].[table]` would work, but than I got the error that the table `'[catalaog].[schema].[schema.table]' could not be found`.  Adding an extra catalog gives a similar error: `'[catalog].[schema].[catalog.schema.table]' can not be found`.

When looking at the ServiceControl unit test code I found out that the correct way of specifying the subscriptions table when in another schema or catalog is as follows `table@schema@catalog`.  I would have been nice to find that in the documentation and not need to look in the code.
